_topic_maps/_topic_map.yml

```diff
@@ -157,7 +157,7 @@ Topics:
     File: installing-aws-private
   - Name: Installing a cluster on AWS into a government region
     File: installing-aws-government-region
-  - Name: Installing a cluster on AWS into a Top Secret Region
+  - Name: Installing a cluster on AWS into a Secret or Top Secret Region
```

Comment:
Hello @mjpytlak @patrickdillon 

I am bit confused by these names:
* `Top Secret Region` == `Commercial Cloud Services Regions` == `C2S Regions`
* `Secret Region` == `Secret Commercial Cloud Services Regions` == `SC2S Regions`

Is it right? 

It looks like we only use `Top Secret Region`/`Secret Region` in the title and TOC, in other places we use `C2S`/`SC2S`, is this intentional?

Reply:
> is this intentional?

Only in the sense that I was following the existing format in [4.10](https://docs.openshift.com/container-platform/4.10/installing/installing_aws/installing-aws-secret-region.html). Open to suggestions. Can you elaborate on what you found confusing?

Reply:
>  Can you elaborate on what you found confusing?

Just curious, my thought was, per security level, looks like `Secret Commercial` > `Commercial ...`, so `Secret Commercial  ... ` should be `Top Secret Region`, and `Commercial ...` should be `Secret Region`. 

But since we have fully reviewed in   https://github.com/openshift/openshift-docs/pull/39769, I agree with current description.

Reply:
Just wanted to give a little bit of insight on C2S. Commercial Cloud Services (C2S) is the program allowing commercial cloud offerings for IC Agencies. The program has three flavors: Unclassified, Secret, and Top Secret. The official abbreviations for them are UC2S, SC2S, TC2S.

The first classified region stood up was TC2S. Since implementation of the C2S program and initial region took so long to be completed, SC2S was delayed several years. During this time, it became common practice to simply refer to TC2S as C2S.

Reply:
Thanks @montaguethomas, good to know these background info.